### ISSUE TITLE
[Bug]: NGINX Logs - [emerg] 11280#12604: empty replacement in...

### ISSUE THREAD

**Issue by ericmp33** [beyondcode/herd-community#514]
### Operating system version

Windows 11

### System architecture

Intel (x86)

### Herd Version

1.0.0

### PHP Version

PHP 8.3.4 (cli) (built: Mar 13 2024 11:42:47) (NTS Visual C++ 2019 x64)

### Bug description

Just installed Herd, haven't configured anything.
I created a new fresh Laravel project and then I tried to start the nginx service but I can't. In the NGNIX Logs, I see this:
```php
2024/03/26 17:57:44 [emerg] 11280#12604: empty replacement in C:/Users/eric/.config/herd/config/valet/Nginx/example-app.test.conf:27
```

### Steps to reproduce

1. Install Herd (fresh)
2. Install a new Laravel project & run `herd link; herd secure` (not sure if this step is actually needed)
4. Try to start all services

### Relevant log output

![image](https://github.com/beyondcode/herd-community/assets/60661635/780f33c1-576d-4d65-894e-9bf2a3d26dc5)

**Comment by mpociot** [comment]
Can you share the content of the configuration file?
`C:/Users/eric/.config/herd/config/valet/Nginx/example-app.test.conf`

**Comment by AlexVanSteenhoven** [comment]
Probably has this to do with https://github.com/beyondcode/herd-community/issues/529#issuecomment-2022986799

**Comment by ericmp33** [comment]
@mpociot - here you have the content of the file `C:/Users/eric/.config/herd/config/valet/Nginx/example-app.test.conf`:
```
server {
    listen 127.0.0.1:80;
    #listen 127.0.0.1:80; # valet loopback
    server_name example-app.test www.example-app.test *.example-app.test;
    return 301 https://$host$request_uri;
}

server {
    listen 127.0.0.1:443 ssl;
    #listen VALET_LOOPBACK:443 ssl; # valet loopback
    server_name example-app.test www.example-app.test *.example-app.test;
    root /;
    charset utf-8;
    client_max_body_size 512M;
    http2 on;

    location ~* /41c270e4-5535-4daa-b23e-c269744c2f45/([A-Z]+:)(.*) {
        internal;
        alias $1;
        try_files $2 $2/;
    }

    ssl_certificate "C:/Users/eric/.config/herd/config/valet/Certificates/example-app.test.crt";
    ssl_certificate_key "C:/Users/eric/.config/herd/config/valet/Certificates/example-app.test.key";

    location / {
        rewrite ^ "" last;
    }

    location = /favicon.ico { access_log off; log_not_found off; }
    location = /robots.txt  { access_log off; log_not_found off; }

    access_log off;
    error_log "C:/Users/eric/.config/herd/Log/nginx-error.log";

    error_page 404 "";

    location ~ [^/]\.php(/|$) {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;

        # Herd specific configuration, do not change
        fastcgi_pass $herd_sock;

        fastcgi_index "";
        include fastcgi_params;
        fastcgi_param SCRIPT_FILENAME "";
        fastcgi_param HERD_HOME "C:/Users/eric/.config/herd";
        fastcgi_param PATH_INFO $fastcgi_path_info;
    }

    location ~ /\.ht {
        deny all;
    }
}

server {
    listen 127.0.0.1:60;
    #listen 127.0.0.1:60; # valet loopback
    server_name example-app.test www.example-app.test *.example-app.test;
    root /;
    charset utf-8;
    client_max_body_size 128M;

    add_header X-Robots-Tag 'noindex, nofollow, nosnippet, noarchive';

    location ~* /41c270e4-5535-4daa-b23e-c269744c2f45/([A-Z]+:)(.*) {
        internal;
        alias /;
        try_files $uri $uri/;
    }

    location / {
        rewrite ^ "" last;
    }

    location = /favicon.ico { access_log off; log_not_found off; }
    location = /robots.txt  { access_log off; log_not_found off; }

    access_log off;
    error_log "C:/Users/eric/.config/herd/Log/nginx-error.log";

    error_page 404 "";

    location ~ [^/]\.php(/|$) {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_pass $herd_sock;
        fastcgi_index "";
        include fastcgi_params;
        fastcgi_param SCRIPT_FILENAME "";
        fastcgi_param HERD_HOME "C:/Users/eric/.config/herd";
        fastcgi_param PATH_INFO $fastcgi_path_info;
    }

    location ~ /\.ht {
        deny all;
    }
}
```

> Did you run herd secure while the Herd desktop application was running? This error could occur if the app is not open, while running herd secure

Not sure about that, I don't remember.

**Comment by mpociot** [comment]
Fixed in version 1.0.1 
https://herd.laravel.com/changelog/windows#101
